ports_module/cortex_a7/gnu/inc/tx_port.h

```diff
@@ -273,17 +276,9 @@ typedef unsigned short                          USHORT;
 /* Determine if the ARM architecture has the CLZ instruction. This is available on
```

Comment:
Since the code is removed, how about removing this comment as well?

Reply:
accepted, resolved.